concert/experiments/imaging.py

```diff
@@ -327,45 +343,22 @@ async def stop_sample_exposure(self):
         raise NotImplementedError()
 
 
-class SpiralTomography(ContinuousTomography):
-    """
-    Spiral Tomography
-
-    This implements a helical acquisition scheme, where the sample is translated perpendicular to the beam while the
-    sample is rotated and the projections are recorded.
-    """
-    vertical_velocity = Quantity(q.mm / q.s)
+class Spiral(Parameterizable):
```

Comment:
Shouldn't this be called `SpiralMixin` actually?

Reply:
Yes